### PR TITLE
#166840998 Configure ESlint

### PR DESCRIPTION
**Pull request by Cavdy**
#### What does this PR do?
Configure the project with ESlint

#### Description of Task to be completed?
- `npm i eslint eslint-config-airbnb-base eslint-plugin-import -D`
- Create .eslintrc.js file and added the configuration

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166840998](https://www.pivotaltracker.com/story/show/166840998)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Comment by chokonaira** [comment]
Looks good

**Review by chokonaira — APPROVED** [review]
Looks good

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }],
+    "require-jsdoc": ["error", {
+      "require": {
+        "FunctionDeclaration": true,
+        "MethodDefinition": true,
+        "ClassDeclaration": true
+      }
+    }]
+  }
+}
```
Comment (codeBlock-1984) [inline .eslintrc.js:37]: Nice job Cavdy. Please add a new line here, and it's good to go.

**Comment by Alpha1202** [comment]
Looks good to me. Nice one

**Review by czarjulius — COMMENTED** [review]
Awesome. Good job

**Review by chokonaira — APPROVED** [review]
good

**Review by czarjulius — APPROVED** [review]

**Review by codeBlock-1984 — APPROVED** [review]

**Review by phembarl — APPROVED** [review]

**Pull request merged**